lib/src/main/java/uk/gov/di/ipv/cri/passport/library/error/RedirectErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.di.ipv.cri.passport.library.error;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.minidev.json.JSONObject;
```

Comment:
Think this might be the wrong JSONObject class we want to use. Should this be the nimbus JSONObject instead?